tests/test-bundler.js

```diff
@@ -32,6 +32,6 @@ const testsToRun = testsContext.keys().filter(inManifest)
 
 // require all `src/**/*.js` except for `main.js` (for isparta coverage reporting)
 if (__COVERAGE__) {
-  const componentsContext = require.context('../src/', true, /^((?!main).)*\.js$/)
+  const componentsContext = require.context('../src/', true, /^((?!main|reducers).)*\.js$/)
```

Comment:
If this was just to get codecov to be happy it's not necessary, since I can just merge anyways.